app/src/components/Map/LayerLibrary.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import LayerItem from 'components/Map/LayerItem';
+
+import LayerLibraryStyles from 'styles/components/map/c-layer-management.scss';
+import LayerListStyles from 'styles/components/map/c-layer-list.scss';
+
+class LayerLibrary extends Component {
+
+  render() {
+    if (!this.props.layers) return null;
+
+    const library = [];
+
+    if (this.props.layers.length > 0) {
+      this.props.layers.forEach((layer, i) => {
+        library.push(<LayerItem
```

Comment:
I'm not sure it's a good idea to reuse LayerItem. In the end we will stuff this component with `if (!this.props.layerLibraryDisplay)`, callbacks will not be the same, it will become an unreadable mess: because noth "Layers" fulfill a completely different functionality.
In fact I don't quite understand the whole AddLayer flow, the layers here should just be layer titles added to the main layers thing, we need to discuss with Dani.